packages/contracts-bedrock/src/dispute/lib/Errors.sol

```diff
@@ -137,3 +137,6 @@ error UnregisteredGame();
 
 /// @notice Thrown when attempting to set an anchor state using an invalid game result.
 error InvalidGameStatus();
+
+/// @notice Thrown when attempting to initialize an anchor state root for an existing game type.
```

Comment:
nit: natspec should say more about what the error means, rather than what code path it occurs in.